static/css/shot-index.scss

```diff
@@ -105,6 +105,10 @@ h1 {
     color: $dark-default;
   }
 
+  a {
+    color: $dark-default;
```

Comment:
@niharikak101 Hmm, it seems funny that the style above this, `.default-color-scheme & a`, isn't getting applied.

Looking at the git history for the shot index page view, it looks like the `default-color-scheme` class may have been [accidentally removed](https://github.com/mozilla-services/screenshots/commit/1efbd956d6397be8c44887d8e47c81a57d981e6f#diff-d45944c551eead471a128c9a716e2a79L34).

And, after trying it out locally, it looks like adding the `default-color-scheme` classname to the landing page fixes the footer color issues on that page, too.

Would you mind removing your changes and adding that classname to the outermost divs in the views for the shotindex and homepage? You can just force-push your changes; ask if you have any questions.